### PR TITLE
smart text support and sdk updates

### PR DESCRIPTION
**Pull request by acantarero**
- add support for smart text
- integrate android sdk v1.2.1
- integrate ios sdk v2.7.0

### CODE REVIEW

File: android/src/main/java/com/nami/reactlibrary/NamiUtil.kt [hunk 1]
```diff
@@ -208,9 +210,6 @@ fun NamiSKU.toSkuDict(): WritableMap {
         SubscriptionPeriod.HALF_YEAR -> {
             "half_year"
         }
-        SubscriptionPeriod.DAY -> {
```
Comment (acantarero) [inline android/src/main/java/com/nami/reactlibrary/NamiUtil.kt:211]: why is this removed?

Comment (KiGi) [reply android/src/main/java/com/nami/reactlibrary/NamiUtil.kt:211]: "Day" no longer exists in Android SDK.

**Review by KiGi — COMMENTED** [review]
Looks good!
